pom.xml

```diff
@@ -24,14 +24,9 @@
 
   <developers>
     <developer>
-      <name>Patrick Fink</name>
-      <email>mail@pfink.de</email>
-      <organization>openHAB</organization>
-      <organizationUrl>http://www.openHAB.org</organizationUrl>
-    </developer>
-    <developer>
-      <name>Kai Kreuzer</name>
-      <email>kai@openhab.org</email>
+      <id>openhab</id>
+      <name>openHAB</name>
+      <email>info@openhab.org</email>
       <organization>openHAB</organization>
       <organizationUrl>http://www.openHAB.org</organizationUrl>
```

Comment:
Should we change that to `https` while touching it?

Reply:
I think so

Reply:
done.